src/api/app/controllers/webui/user_controller.rb

```diff
@@ -59,6 +59,7 @@ def show
     return unless User.current == @displayed_user
     @is_displayed_user = User.current == @displayed_user
     @patchinfos = @displayed_user.involved_patchinfos
+    @account_edit_link = CONFIG['proxy_auth_account_page'] unless CONFIG['proxy_auth_account_page'].blank?
```

Comment:
See line 37 in src/api/app/views/webui/user/show.html.haml

```
- if @account_edit_link
  = link_to sprited_text('user_edit', 'Edit your account'), @account_edit_link
```

Drop the ``blank?`` here and do it instead only in the view. And btw: I would prefer using ``if present?`` and avoid using unless when possible.